Makefile

```diff
@@ -15,10 +15,19 @@
 IMAGE=amazon/aws-efs-csi-driver
 VERSION=0.1.0
 
+FLAG_COMMIT=-X github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/info.commitSha=`git log --pretty=format:'%H' -n 1`
+FLAG_DRIVER=-X github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/driver.vendorVersion=${VERSION} -X github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/info.driver=${VERSION}
+FLAG_ARCH=-X github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/info.arch=`uname -m`
```

Comment:
You can use runtime.GOOS and runtime.GOARCH to figure out the os/arch is a consistent why